.github/workflows/ci.yaml

```diff
@@ -30,6 +30,7 @@ jobs:
 
       - name: Install dependencies
         run: |
+          pip install --upgrade pip
```

Comment:
This is because of a blocking library vuln check complaining about something that can happen when you pip install from a mercurial repo on old versions of pip 🙄